api/layer/layer.go

```diff
@@ -472,6 +280,23 @@ func (n *layer) EphemeralKey() *keys.PublicKey {
 	return n.anonKey.Key.PublicKey()
 }
 
+func (n *layer) Initialize(ctx context.Context, c Notificator) error {
```

Comment:
Was it necessary to provide `Notificator` directly into `Initialize()` instead of keeping it in config? 
It seems that in the end you still do `n.ncontroller = c`.

Is it possible to call `Initialize()` twice during runtime?

Reply:
Actually this function isn't necessary for current PR (it was written when I thought we have to handle ticks for locking objects). But I would like to preserve this function since another [PR](https://github.com/nspcc-dev/neofs-s3-gw/pull/371) is based on it.

The function `Initialize()` should be called once (at least until we support adding/removing handlers for ticks in runtime). That's why I provide `Notificator` directly into this function - I wanted to avoid introducing new flag in `layer` structure that allows to know if the layer is initialized.